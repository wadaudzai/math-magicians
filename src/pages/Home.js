import React from 'react';

const Home = () => (
  <section className="home">
    <h2>Welcome to our page.</h2>
    <p>This is a page that is built to display the home page of our website.</p>
    <p>This is a page that is built to display the home page of our website.</p>
    <p>This is a page that is built to display the home page of our website.</p>
    <p>This is a page that is built to display the home page of our website.</p>
    <p>This is a page that is built to display the home page of our website.</p>
    <p>This is a page that is built to display the home page of our website.</p>
    <p>This is a page that is built to display the home page of our website.</p>
  </section>
);

export default Home;
