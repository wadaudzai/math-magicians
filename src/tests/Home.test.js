import React from "react";
import renderer from 'react-test-renderer';
import Home from '../pages/Home';

describe('Check Home Component', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Home />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})