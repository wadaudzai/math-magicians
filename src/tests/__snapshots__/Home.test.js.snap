// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Home Component renders correctly 1`] = `
<section
  className="home"
>
  <h2>
    Welcome to our page.
  </h2>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
  <p>
    This is a page that is built to display the home page of our website.
  </p>
</section>
`;
